src/main/java/com/virtudoc/web/controller/UsersChatController.java

```diff
@@ -0,0 +1,59 @@
+package com.virtudoc.web.controller;
+
+
+import com.virtudoc.web.entity.Appointment;
+import com.virtudoc.web.entity.UserAccount;
+import com.virtudoc.web.service.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@RestController
+@CrossOrigin
```

Comment:
Just out of curiosity, is this for CORS? That's weird that it's needed for our application since all communication is supposed to happen on the same domain.

Reply:
Sorry that I meant to remove CrossOrigin. I was just testing some random stuff.